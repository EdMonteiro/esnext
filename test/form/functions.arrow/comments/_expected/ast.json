{
  "type": "Program",
  "body": [
    {
      "type": "ExpressionStatement",
      "expression": {
        "type": "CallExpression",
        "callee": {
          "type": "Identifier",
          "name": "a"
        },
        "arguments": [
          {
            "type": "ArrowFunctionExpression",
            "id": null,
            "params": [],
            "body": {
              "type": "Literal",
              "value": 42,
              "raw": "42"
            },
            "generator": false,
            "expression": false
          }
        ]
      }
    }
  ],
  "sourceType": "module"
}