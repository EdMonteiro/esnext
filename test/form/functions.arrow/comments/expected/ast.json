{
  "type": "Program",
  "body": [
    {
      "type": "ExpressionStatement",
      "expression": {
        "type": "CallExpression",
        "callee": {
          "type": "Identifier",
          "name": "a",
          "range": [
            0,
            1
          ],
          "loc": {
            "start": {
              "line": 1,
              "column": 0
            },
            "end": {
              "line": 1,
              "column": 1
            }
          }
        },
        "arguments": [
          {
            "type": "ArrowFunctionExpression",
            "id": null,
            "params": [],
            "body": {
              "type": "Literal",
              "value": 42,
              "raw": "42",
              "range": [
                88,
                90
              ],
              "loc": {
                "start": {
                  "line": 3,
                  "column": 9
                },
                "end": {
                  "line": 3,
                  "column": 11
                }
              }
            },
            "generator": false,
            "expression": false,
            "range": [
              2,
              93
            ],
            "loc": {
              "start": {
                "line": 1,
                "column": 2
              },
              "end": {
                "line": 4,
                "column": 1
              }
            }
          }
        ],
        "range": [
          0,
          94
        ],
        "loc": {
          "start": {
            "line": 1,
            "column": 0
          },
          "end": {
            "line": 4,
            "column": 2
          }
        }
      },
      "range": [
        0,
        95
      ],
      "loc": {
        "start": {
          "line": 1,
          "column": 0
        },
        "end": {
          "line": 4,
          "column": 3
        }
      }
    }
  ],
  "sourceType": "module",
  "range": [
    0,
    95
  ],
  "loc": {
    "start": {
      "line": 1,
      "column": 0
    },
    "end": {
      "line": 4,
      "column": 3
    }
  }
}